src/bin/rugs_metadata_server.rs

```diff
@@ -46,9 +46,15 @@ struct Config {
 impl Config {
     /// Construct a config from environment variables or sensible defaults
     fn from_env() -> Self {
-        let user_auth = std::env::var("RUGS_USER_AUTH").ok();
+        let user_auth = std::env::var("RUGS_USER_AUTH_FILE")
+            .ok()
+            .and_then(|path| std::fs::read_to_string(path).ok())
```

Comment:
Might be good to add a `.map(|p| p.trim_end())` or `trim_end_matches` with a filter for `\r` and `\n` to strip trailing newlines in case the document has a trailing newline. :)